apps/dashboard/app/models/launcher.rb

```diff
@@ -340,10 +340,17 @@ def add_script_to_form(form: [], attributes: {})
 
     dir = { directory: project_dir }
     attributes[:auto_scripts] = if attributes[:auto_scripts]
-      attributes[:auto_scripts].merge(dir)
-    else
-      dir
-    end
+                                  attributes[:auto_scripts].merge(dir)
+                                else
+                                  dir
+                                end
+
+    # Delete 'value' if it does not exist in the available options
+    # so that projects created from templates don't point to another
+    # user's script
+    script_value = attributes[:auto_scripts]['value']
+    script_options = attributes[:auto_scripts]['options']
+    attributes[:auto_scripts].delete('value') if script_options.none? { |opt| opt.include?(script_value) }
```

Comment:
I feel like I fixed this at some point - or maybe it was the same issue, but with accounts?

Can you let me know how you ran into this issue? Seems like the template should hold correct values (correct for the template) and when it's copied anything that would be a `value` would be correct given the new directory. Or is it a case of relative paths that would work and absolute paths that still point to the template's script?

I'll have to look at how I did it for accounts. It seemed simpler than this, like just re-initializing the smart_attribute so it could reset itself (instead of us resetting it here in this layer).

Reply:
Yea #3401 was how I fixed a similar issue for accounts. Basically a template is made with accounts A, B & C - but when the user copies the templates the account list should be D, E & F, to use for the user's account list and not the template generator's accounts.

Reply:
I ran into this issue when changing the way that `directory` is added to `auto_scripts`. Before, we actually entirely overwrote the attribute with only `directory`, so the old `value` that points to the template's script would get erased there as well.

It might be a relative vs. absolute path case, since the value in the template I was testing with was an absolute path to the template's script, while the only option available was the same script in the new, correct directory. Re-initializing the attribute would probably work here as well.